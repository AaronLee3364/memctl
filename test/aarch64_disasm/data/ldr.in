fffffff0070c0000
f94072e9 f9400268 794023e9 39401909
f8408d2b b9822288 b840440d b9800fe8
39c0252a 79c5c289 58d83868 98747920
