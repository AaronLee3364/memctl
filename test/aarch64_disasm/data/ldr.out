fffffff0070c0000  LDR     X9, [X23, #224]
fffffff0070c0004  LDR     X8, [X19]
fffffff0070c0008  LDRH    W9, [SP, #16]
fffffff0070c000c  LDRB    W9, [X8, #6]
fffffff0070c0010  LDR     X11, [X9, #8]!
fffffff0070c0014  LDRSW   X8, [X20, #544]
fffffff0070c0018  LDR     W13, [X0], #4
fffffff0070c001c  LDRSW   X8, [SP, #12]
fffffff0070c0020  LDRSB   W10, [X9, #9]
fffffff0070c0024  LDRSH   W9, [X20, #736]
fffffff0070c0028  LDR     X8, #0xfffffff007070734
fffffff0070c002c  LDRSW   X0, #0xfffffff0071a8f50
